### DISCUSSION TITLE
Plugins folder permission problem

### DISCUSSION THREAD

**Discussion by wsyong111** [rocksdanister/lively#1660] [Q&A] [answered]
I want to install a plugin, but I can't add files to the Plugins folder (Access is denied)
I have tried to change the permissions, although I have obtained "full control" permissions on the permissions, but I still can't add files
![image](https://user-images.githubusercontent.com/94914969/230303336-8045aa84-dff6-41bf-b2be-e6e8335e5ef3.png)

_I use Google translate to translate_

**Comment by rocksdanister** [comment, marked as answer]
Microsoft Store version does not allow users to modify the folders.

You will need to uninstall and install the installer version:
https://github.com/rocksdanister/lively/wiki/Differences-Between-Distributions#installergithub-release
